superset/assets/stylesheets/superset.css

```diff
@@ -208,4 +208,8 @@ div.widget .slice_container {
 
 .table-condensed input[type="checkbox"] {
     float: left;
+}
+
+.form-inline {
```

Comment:
Thanks for the tip. I just changed the code to target `.sql-toolbar .form-inline`.